content/docs/command-reference/data/status.md

```diff
@@ -0,0 +1,133 @@
+# data status
+
+Show changes in the data tracked by DVC in the workspace.
+
+## Synopsis
+
+```usage
+usage: dvc data status [-h] [-q | -v]
+                       [--granular] [--unchanged]
+                       [--untracked-files [{no,all}]]
+                       [--json]
+```
+
+## Description
```

Comment:
I have rewritten the description as @shcheklein suggested. I have added a description and a small list to explain individual categories.

Also, I have thrown out the mention of `metafiles`, I think we can talk mostly in terms of files and changes here.